schemas/rum/_common-schema.json

```diff
@@ -300,6 +300,28 @@
             }
           }
         },
+        "configuration": {
+          "type": "object",
+          "description": "Subset of the SDK configuration options in use during its execution",
+          "readOnly": true,
+          "required": ["session_sample_rate", "session_replay_sample_rate"],
+          "properties": {
+            "session_sample_rate": {
+              "type": "integer",
+              "description": "The percentage of sessions tracked",
+              "minimum": 0,
+              "maximum": 100,
+              "readOnly": true
+            },
+            "session_replay_sample_rate": {
+              "type": "integer",
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it seems that it should be number to accept floating point numbers (cf [doc](https://json-schema.org/understanding-json-schema/reference/numeric.html)). What about adding a test for this one?
```suggestion
              "type": "number",
              "description": "The percentage of sessions tracked",
              "minimum": 0,
              "maximum": 100,
              "readOnly": true
            },
            "session_replay_sample_rate": {
              "type": "number",
```